runner/buckets.go

```diff
@@ -108,5 +108,13 @@ var (
 			// 0.1s, 0.5s, 1s, 5s, 1m, 5m, 10m, 30m, 1h, 12h, 24h
 			0.1, 0.5, 1, 5, 60, 300, 600, 1800, 3600, 12 * 3600, 24 * 3600,
 		},
+		"tracked_users_hll_bytes": {
+			float64(10 * bytesize.B),  // for hll containing single id = 8B
+			float64(100 * bytesize.B), // for hll containing 10 ids = 80B
+			float64(1 * bytesize.KB),  // for hll containing 100 ids = 800B
+			float64(3 * bytesize.KB),  // for hll containing 300 ids = 2400B
+			float64(6 * bytesize.KB),  // for hll containing 700 ids = 5600B
+			float64(10 * bytesize.KB), // max size for hll with log2m=14, regWidth=5 = 10KB
+		},
```

Comment:
Will need to fine tune the buckets for this one I believe
```suggestion
		},
		"processor_tracked_users_report_gen_seconds": {
		     // 1ms, 10ms, 50ms, 100ms, 0.5s, 1s
		}
```

Reply:
yes this is in micro seconds, I will update this.

Reply:
For now we can keep this metric for some time, we can decide and remove later after checking in prod. May be in next server release?